src/EnumSet.php

```diff
@@ -198,11 +190,75 @@ public function count()
         $cnt = 0;
         $ord = 0;
         do {
-            if ($this->bitset & (1 << $ord++)) {
+            if ($this->getBit($ord++)) {
                 ++$cnt;
             }
         } while ($ord !== $this->ordinalMax);
 
         return $cnt;
     }
+
+    /**
+     * Get the binary bitset
+     * @return string Returns the binary bitset in big-endian order
+     */
+    public function getBitset() {
+        return strrev($this->bitset);
+    }
+
+    /**
+     * Set the bitset.
+     * NOTE: It resets the current position of the iterator
+     * @param string $bitset The binary bitset in big-endian order
+     * @return void
+     */
+    public function setBitset($bitset) {
+        if (!is_string($bitset)) {
+            throw new InvalidArgumentException("bitset must be a string");
+        }
+
+        $bitset = strrev($bitset);
+        $size   = ceil($this->ordinalMax / 8);
+        $sizeIn = strlen($bitset);
+
+        if ($sizeIn < $size) {
+            //add "\0" if the given bitset is not long enough
+            $bitset .= str_repeat("\0", $size - $sizeIn);
+        } elseif ($sizeIn > $size) {
+            $bitset = substr($bitset, 0, $size);
+        }
+
+        $this->bitset = $bitset;
+
+        $this->rewind();
+    }
+
+    /**
+     * get a bit at the given ordinal
+     * @param $ordinal int  number of bit to get
+     * @return boolean
+     */
+    private function getBit($ordinal) {
+        return (ord($this->bitset[(int)($ordinal/8)]) & 1 << ($ordinal % 8))!==0;
```

Comment:
Micro optimization: What about returning the integer value instead of a boolean as this method is only for comparison this should work similar using direct comparison (`if ($this->getBit($ord))`) but reduces one internally cast to boolean.